draft-ietf-taps-interface.md

```diff
@@ -1554,66 +1554,98 @@ Security parameters and callbacks are partitioned based on their place in the li
+- One or more certificate bundles identifying the Local Endpoint, whether
+  as a server certificate or a client certificate. Multiple bundles may
+  be provided to allow selection among different protocol stacks that may
+  require differently formatted bundles. The form and format of the
+  certificate bundle is implementation-specific. Note that if the private
+  keys associated with a bundle are not available, e.g., since they are stored in hardware
+  security modules (HSMs), handshake callbacks must be used. See below for details.
 
 ~~~
-SecurityParameters.Set(alpn, "h2")
+SecurityParameters.Set(server-certificate, myCertificateBundle[])
+SecurityParameters.Set(client-certificate, myCertificateBundle[])
 ~~~
 
-- Local identity, certificates, and private keys: Used to perform private key operations and prove one's
-identity to the Remote Endpoint. (Note, if private keys are not available, e.g., since they are
-stored in hardware security modules (HSMs), handshake callbacks must be used. See below for details.)
+- One or more certificate chains to use as pinned server
+  certificates, such that connection will fail if the presented server
+  certificate does not match one of the  supplied pinned certificates.
+  The form and format of the certificate chain is implementation-specific.
 
 ~~~
-SecurityParameters.Set(identity, myIdentity)
-SecurityParameters.Set(server-certificate, myCertificate)
-SecurityParameters.Set(client-certificate, myCertificate)
-SecurityParameters.Set(key-pair, myPrivateKey, myPublicKey)
+SecurityParameters.Set(pinned-server-certificate, yourCertificateChain[])
 ~~~
 
-- Supported algorithms: Used to restrict what parameters are used by underlying transport security protocols.
-When not specified, these algorithms should use known and safe defaults for the system. Parameters include:
-ciphersuites, supported groups, and signature algorithms. These parameters take a collection of supported algorithms as parameter.
+- Application-layer protocol negotiation (ALPN) values: used to indicate which
+  application-layer protocols are negotiated by the security protocol layer.
+  See {{!ALPN=RFC7301}} for definition of the ALPN field. Note that the Transport
+  Services System can provide ALPN values automatically, based on
+  the protocols being used, if not explicitly specified by the application.
 
 ~~~
-SecurityParameters.Set(supported-group, "secp256r1")
-SecurityParameters.Set(ciphersuite, "TLS_AES_128_GCM_SHA256")
-SecurityParameters.Set(signature-algorithm, "ecdsa_secp256r1_sha256")
+SecurityParameters.Set(alpn, "h2")
 ~~~
 
-- Pre-Shared Key import: Used to install pre-shared keying material established
-out-of-band. Each pre-shared keying material is associated with some identity that typically identifies
-its use or has some protocol-specific meaning to the Remote Endpoint.
+- Supported algorithms, ciphersuites, groups: Used to restrict what parameters
```

Comment:
```suggestion
- Supported groups, ciphersuites, algorithms: Used to restrict what parameters
```
I think this list should be in the same order as the three example commands below. You could also go the other way round - change the order of commands. I don't think it matters, but it's a bit weird to have this in a different sequence here.